src/specialOffer/SpecialOffer.style.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components'
+
+import { responsiveBreakpoints, space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
+
+export const StyledSpecialOffer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`
+
+export const StyledSpecialOfferContent = styled.div`
+  flex: 1;
+  align-items: flex-start;
+  max-width: 50%;
+
+  @media (${responsiveBreakpoints.isMediaSmall}) {
+    max-width: 100%;
+    min-width: 100%;
+  }
```

Comment:
I'm not sure it's the best way to do this. Don't hesitate to propose something else here 😁

Reply:
I'm no expert but I would have done it like this as well 😛

Reply:
I would rather use mobile online declaration